diff --git a/geoportal/geomapfish_geoportal/static/apihelp/index.html.tmpl b/geoportal/geomapfish_geoportal/static/apihelp/index.html.tmpl
index 33e7852..e87172a 100644
--- a/geoportal/geomapfish_geoportal/static/apihelp/index.html.tmpl
+++ b/geoportal/geomapfish_geoportal/static/apihelp/index.html.tmpl
@@ -77,7 +77,7 @@ window.onload = function() {
         <h2>A map</h2>
         <div id="map1" class="map"></div>
         <pre data-language="javascript">
-const map1 = new geomapfish.Map({
+const map1 = new geomapfishapp.Map({
     div: 'map1', // id of the div element to put the map in
     zoom: 4,
     center: [2544500, 1210100]
@@ -90,7 +90,7 @@ const map1 = new geomapfish.Map({
         <h2>A map with a marker on its center</h2>
         <div id="map2" class="map"></div>
         <pre data-language="javascript">
-const map2 = new geomapfish.Map({
+const map2 = new geomapfishapp.Map({
     div: 'map2',
     zoom: 7,
     backgroundLayers: ['OSM map'],
@@ -105,7 +105,7 @@ map2.addMarker();
         <h2>A map with several custom markers</h2>
         <div id="map3" class="map"></div>
         <pre data-language="javascript">
-const map3 = new geomapfish.Map({
+const map3 = new geomapfishapp.Map({
     div: 'map3',
     zoom: 7,
     center: [2544500, 1210100]
@@ -133,7 +133,7 @@ map3.addMarker({
         <h2>A map with a subset of overlays</h2>
         <div id="map4" class="map"></div>
         <pre data-language="javascript">
-const map4 = new geomapfish.Map({
+const map4 = new geomapfishapp.Map({
     div: 'map4',
     zoom: 0,
     center: [2590000, 1170000],
@@ -147,7 +147,7 @@ const map4 = new geomapfish.Map({
         <h2>A map with some additional controls</h2>
         <div id="map5" class="map"></div>
         <pre data-language="javascript">
-const map5 = new geomapfish.Map({
+const map5 = new geomapfishapp.Map({
     div: 'map5',
     zoom: 3,
     center: [2544500, 1210100],
@@ -168,7 +168,7 @@ const map5 = new geomapfish.Map({
         <br />
         <div id="map6" class="map"></div>
         <pre data-language="javascript">
-const map6 = new geomapfish.Map({
+const map6 = new geomapfishapp.Map({
     div: 'map6',
     addMiniMap: true,
     miniMapExpanded: false
@@ -189,7 +189,7 @@ button2.onclick = function() {
         <h2>Recenter the map on objects</h2>
         <div id="map7" class="map"></div>
         <pre data-language="javascript">
-const map7 = new geomapfish.Map({
+const map7 = new geomapfishapp.Map({
     div: 'map7',
     layers: ['polygon']
 });
@@ -211,7 +211,7 @@ map7.recenterOnObjects(
         <br />
         <div id="map8" class="map"></div>
         <pre data-language="javascript">
-const map8 = new geomapfish.Map({
+const map8 = new geomapfishapp.Map({
     div: 'map8'
 });
 map8.addCustomLayer('text', 'My custom txt layer', '../static/${CACHE_VERSION}/apihelp/data.txt', {
@@ -228,7 +228,7 @@ map8.addCustomLayer('text', 'My custom txt layer', '../static/${CACHE_VERSION}/a
         <div id="map9search" style="width: 400px"></div>
         <div id="map9" class="map"></div>
         <pre data-language="javascript">
-const map9 = new geomapfish.Map({
+const map9 = new geomapfishapp.Map({
     div: 'map9',
     zoom: 3,
     center: [2544500, 1210100],
