diff --git a/.github/workflows/main.yaml b/.github/workflows/main.yaml
index 18461df..be9cd32 100644
--- a/.github/workflows/main.yaml
+++ b/.github/workflows/main.yaml
@@ -28,7 +28,7 @@ jobs:
 
       - uses: actions/setup-python@v6
         with:
-          python-version: '3.13'
+          python-version: '3.14'
       - run: python3 -m pip install --requirement=ci/requirements.txt
       - name: Environment information
         run: c2cciutils-env
diff --git a/env.project b/env.project
index a10e420..0729e6e 100644
--- a/env.project
+++ b/env.project
@@ -40,10 +40,6 @@ TILEGENERATION_SQS_QUEUE=<queue_name>
 TILEGENERATION_S3_BUCKET=<bucket_name>
 TILEGENERATION_AZURE_CONTAINER=<container_name>
 
-# Should be filed with a strong secret, with e.g. `pwgen --secure 16`
-# TILECLOUD_CHAIN_SESSION_SECRET=
-# TILECLOUD_CHAIN_SESSION_SALT=
-
 # For production
 # FRONT_INNER_PORT=80
 # FRONT_CONFIG=haproxy
