diff --git a/Makefile b/Makefile
index b098488..e5996a9 100644
--- a/Makefile
+++ b/Makefile
@@ -56,11 +56,12 @@ secrets: ## Decrypt the secrets.tar.bz2.gpg file
 
 .PHONY: acceptance-init
 acceptance-init: ## Initialize the acceptance tests
-	docker-compose --file=docker-compose.yaml --file=docker-compose-db.yaml up -d
-	docker-compose exec -T geoportal wait-db
+	docker-compose --file=docker-compose.yaml --file=docker-compose-db.yaml up -d db tools
+	docker-compose exec -T tools wait-db
 	docker-compose exec -T tools psql --command="DROP EXTENSION IF EXISTS postgis CASCADE"
 	scripts/db-restore --docker-compose-file=docker-compose.yaml --docker-compose-file=docker-compose-db.yaml \
 		--arg=--clean --arg=--if-exists --arg=--verbose $(DUMP_FILE)
+	docker-compose --file=docker-compose.yaml --file=docker-compose-db.yaml up -d
 
 .PHONY: acceptance
 acceptance: ## Run the acceptance tests
