diff --git a/tilegeneration/config.yaml.tmpl b/tilegeneration/config.yaml.tmpl
index 9d8c379..a64d766 100644
--- a/tilegeneration/config.yaml.tmpl
+++ b/tilegeneration/config.yaml.tmpl
@@ -39,15 +39,17 @@ defaults:
   layer: &layer
     type: wms
     grid: swissgrid_05
-    # The minimum resolution to seed, useful to use with mapcache, optional.
+    # the minimum resolution to seed, useful to use with mapcache, optional.
     # min_resolution_seed: 1
     # the URL of the WMS server to used
-    url: ${MAPSERVER_URL}
-    # Set the headers to get the right virtual host, and don't get any cached result
+    url: ${MAPSERVER_URL}/mapserv_proxy
+    # don't get any cached result
     headers:
-${DISABLE_LOCAL}      Host: '${VISIBLE_WEB_HOST}'
       Cache-Control: no-cache, no-store
       Pragma: no-cache
+    # get the right Mapfile
+    params:
+      MAP: MapServer
     # file name extension
     extension: png
     # legend file name extension
